### ISSUE TITLE
Add/Update - Variable Product with regular_price

### ISSUE THREAD

**Issue by hikarushin** [XiaoFaye/WooCommerce.NET#162]
When adding or updating variable product. I received below error. 

```
variation = (new Variation
{
	sku = "TEST-VAR",
	//image = varImage.Count == 0 ? null : varImage,
	manage_stock = true,
	regular_price = 15,//(decimal)oHeader.Rows[_Header]["Price"],
	attributes = varAtt
});
```

`var newProdVar = await wc.Product.Variations.Update(WooId, variation, Convert.ToInt32(newProd.id));`
```
{
  "code": "rest_invalid_param",
  "message": "Invalid parameter(s): regular_price",
  "data": {
    "status": 400,
    "params": {
      "regular_price": "regular_price is not of type string."
    }
  }
}
```

**Comment by goguvi** [comment]
Take a look at #147

**Comment by hikarushin** [comment]
Solved #156
